### DISCUSSION TITLE
What is the difference between Servo and a Browser?

### DISCUSSION THREAD

**Discussion by patientplatypus6** [servo/servo#29146] [General]
This may be a dumb question, but I want to know what the difference is between Servo and a browser. 

Here are some examples of sites that I went to using Servo. They load, but the elements are broken or there are issues with loading the page properly. 

Here's `https://www.wsj.com`

<img width="517" alt="Screenshot 2022-12-15 at 12 34 51 PM" src="https://user-images.githubusercontent.com/60800027/207961547-61e8ae00-944c-4dcb-aee3-3f56d6629c46.png">

Here's `https://www.yahoo.com`

<img width="729" alt="Screenshot 2022-12-15 at 12 35 54 PM" src="https://user-images.githubusercontent.com/60800027/207961730-32d8451f-7601-4860-ab91-93c7ab1c74ea.png">

Here's `https://www.reddit.com`

<img width="732" alt="Screenshot 2022-12-15 at 12 36 27 PM" src="https://user-images.githubusercontent.com/60800027/207961830-d9568265-1fed-4a49-8aef-4332d19e0e22.png">

Can someone comment on what else is needed in order to turn this into a browser? You can't scroll and you also can't click on any of the hyperlinks. Many of the images won't load. 

Again, maybe this is a stupid question, but is this repo supposed to be a proof of concept for how a finished engine should look or is it assumed there is a bunch of heavy lifting that isn't shown? And if so, where can I find more information on making this work more like a browser? 

Thanks.

**Comment by jdm** [comment]
Real web content is complex. Right now, Servo is closer to a proof-of-concept in many ways, because a lot more effort is required to get it consistently rendering complex websites.
